src/Analyzer.Core.UnitTests/templates/SimpleNestedFail.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    }
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Resources/deployments",
+      "apiVersion": "2019-10-01",
+      "name": "nestedTemplate",
+      "resourceGroup": "my-rg",
+      "properties": {
+        "mode": "Incremental",
+        "template": {
+          "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+          "contentVersion": "1.0.0.0",
+          "parameters": {},
+          "variables": {},
+          "resources": [
+            {
+              "apiVersion": "2019-08-01",
+              "type": "Microsoft.Web/sites",
+              "kind": "linux",
+              "name": "linuxKind",
+              "location": "[parameters('location')]",
+              "properties": {
+              }
+            },
+            {
+              "apiVersion": "2019-08-01",
+              "type": "Microsoft.Web/sites",
+              "kind": "api",
+              "name": "withoutSpecifyingProperties",
+              "location": "[parameters('location')]",
+                                                        // Blank line to test line numbeer matching with poorly-formatted json templates
```

Comment:
```suggestion
```

This is not necessary anymore after changing the way of calculating line numbers